src/tool/interpreter/local_interpreter.py

```
@@ -0,0 +1,1623 @@
+def evaluate_lambda(
+    lambda_expression: ast.Lambda,
+    state: dict[str, Any],
+    static_tools: dict[str, Callable],
+    custom_tools: dict[str, Callable],
+    authorized_imports: list[str],
+) -> Callable:
+    args = [arg.arg for arg in lambda_expression.args.args]
+
+    def lambda_func(*values: Any) -> Any:
+        new_state = state.copy()
+        for arg, value in zip(args, values):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)